iothub/service/src/DigitalTwin/generateCode.ps1

```diff
@@ -0,0 +1,124 @@
+﻿# Copyright (c) Microsoft. All rights reserved.
```

Comment:
I may be a bit out of context here, can you quickly help me understand why aren't we using the autorest customization capabilities?

Reply:
@abhipsaMisra

Reply:
@azabbasi , @barustum - autorest customization capabilities aren't available on autorest v2.xx (which is the latest version that is available on npm). It is available only on later versions of autorest, but that is restricted to the client being a part of the track 2 repo directly.